tsdb/exemplar_test.go

```diff
@@ -76,6 +77,19 @@ func TestAddExemplar(t *testing.T) {
 	e3.Value = 0.3
 	err = es.AddExemplar(l, e3)
 	require.Equal(t, err, storage.ErrOutOfOrderExemplar)
+
+	e4 := exemplar.Exemplar{
+		Labels: labels.Labels{
+			labels.Label{
+				Name:  "a",
+				Value: strings.Repeat("b", ExemplarMaxLabelSetLength),
+			},
+		},
+		Value: 0.1,
+		Ts:    2,
+	}
+	err = es.AddExemplar(l, e4)
+	require.Equal(t, err, storage.ErrExemplarLabelLength)
```

Comment:
```suggestion
	require.Equal(t, storage.ErrExemplarLabelLength, err)
```

Reply:
also we could inline `es.AddExemplar(l, e4)` really